source/reviews/standard.txt

```diff
@@ -0,0 +1,90 @@
+======================
+The Standard of Review
+======================
+
+.. _pr-standard:
+
+Purpose
+-------
+
+The primary purpose of a review is to make sure that the overall health of MongoDB's
+corpus is improving over time. All of the tools and processes of review are
+designed to this end.
+
+In order to accomplish this, a series of trade-offs have to be balanced.
+
+First, authors must be able to **make progress** on their tasks. If you never
+submit an improvement to the corpus, then the corpus never improves. Also,
+if a reviewer makes it very difficult for *any* change to go in, then authors
```

Comment:
I'm not sure whether this means this is difficult due to some inaccuracy or refusal by the reviewee, or lack of solutions presented by the reviewer. I think you've recommended providing solutions as the reviewer, so it would be useful to identify whether this refers to specifically.